vortex-array/src/arrays/varbin/compute/is_constant.rs

```diff
@@ -9,8 +9,11 @@ impl IsConstantKernel for VarBinVTable {
     fn is_constant(
         &self,
         array: &VarBinArray,
-        _opts: &IsConstantOpts,
+        opts: &IsConstantOpts,
     ) -> VortexResult<Option<bool>> {
+        if opts.is_constant() {
```

Comment:
I wonder if we should instead be writing something like `if opts.cost <= Cost::Constant`? Will we ever add more? Feel free to say no

Reply:
Actually, this would solve the `is_constant` naming problem too